src/SupportsChrome.php

```diff
@@ -42,4 +39,22 @@ public static function stopChromeDriver()
             static::$chromeProcess->stop();
         }
     }
+
+    protected static function initChromeDriver(){
+        static::$chromeProcess = (new ProcessBuilder())
+            ->setPrefix(realpath(__DIR__.'/../bin/chromedriver-' . static::getOSSuffix() . '.exe'))
```

Comment:
Good catch! I have moved it to `getOSSuffix()` so it should work for non WINNT platforms